<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>天气：</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="manageBt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>75</width>
      <height>140</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>140</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>140</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>管
理
员</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraDisplay">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevPage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPage">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>240</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose1_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose3_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose4_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose2_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose5_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>75</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车
位
10</string>
    </property>
   </widget>
   <widget class="QLabel" name="carNumLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>剩余车位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopTimeLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>停车时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="priceLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>停车费用：</string>
    </property>
   </widget>
   <widget class="QLabel" name="carNum">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stopTime">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="price">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="flashCar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模拟车位退出车库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takePhoto">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>7</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
